bitnami/etcd/templates/NOTES.txt

```diff
@@ -0,0 +1,75 @@
+{{- if contains .Values.service.type "LoadBalancer" }}
+{{- if .Values.allowNoneAuthentication }}
+-------------------------------------------------------------------------------
+ WARNING
+
+    By specifying "service.type=LoadBalancer" and "allowNoneAuthentication=true" you
+    have most likely exposed the Redis service externally without any authentication
+    mechanism.
+
+    For security reasons, we strongly suggest that you switch to "ClusterIP" or
+    "NodePort". As alternative, you can also switch to "usePassword=true"
+    providing a valid password on "password" parameter.
+
+-------------------------------------------------------------------------------
+{{- end }}
+{{- end }}
+
+** Please be patient while the chart is being deployed **
+
+etcd can be accessed via port 2379 on the following DNS name from within your cluster:
```

Comment:
Here you should use the etcd port parameter that you specified in Values.yaml

Reply:
right